pkg/clusterlinker/client.go

```diff
@@ -81,3 +85,33 @@ func (c *Client) Unlink(linkedClusterID string) error {
 
 	return nil
 }
+
+// Links returns the links of a cluster.
+func (c *Client) Links() ([]*Link, error) {
+	resp, err := c.http.Get("/cluster/v1/links")
+	if err != nil {
+		return nil, errors.New("couldn't get linked clusters")
+	}
+
+	defer resp.Body.Close()
+
+	// Common error on clusters < 1.11 and OSS clusters.
+	if resp.StatusCode == 404 {
+		return nil, errors.New("couldn't find linked clusters")
```

Comment:
Why would a specific error be returned on 404s here (when the cluster linker API is not available), and not in other methods? I'd skip this condition altogether.

Reply:
I did this because the error message otherwise is not helpful, an example when using `dcos cluster attach` with a cluster running on 110:
```
cli (DCOS_OSS-3695) $ ./dcos-go cluster link http://cluster.com
Error: invalid character '<' looking for beginning of value
```
This is because we do not have a `dcos.error` when DC/OS returns a 404.